s3-lambda-bedrock-aoss/README.md

```diff
@@ -0,0 +1,94 @@
+# S3 to Lambda to OpenSearch Serverless with Bedrock for Document Embeddings
+
+This patterns can be used to automatically create embeddings using Amazon Bedrock for documents uploaded to an Amazon S3 bucket and store the embeddings along with the document content in an Amazon OpenSearch Serverless index.
+
+This patterns is useful for RAG-based applications that use Amazon S3 as source for the knowledge base for their LLM models. The pattern helps to seamlessly create and store the embeddings for new documents once they are upload to S3.
+
+Learn more about this pattern at Serverless Land Patterns: << Add the live URL here >>
```

Comment:
Please add URL.. you can take reference from any other pull requests

Reply:
fixed